tests/Feature/Engine/EngineTestCase.php

```diff
@@ -259,4 +265,20 @@ protected function dispatchJob()
         $instance = $job['token'] ? $job['token']->getInstance() : null;
         return $job ? $job['element']->execute($job['eventDefinition'], $instance) : null;
     }
+
+    protected function getFactory() {
+
+        $mappings = [
+            \ProcessMaker\Nayra\Contracts\Bpmn\ActivityInterface::class => \ProcessMaker\Models\Activity::class,
```

Comment:
Declare the classes (use ProcessMaker\Nayra\Contracts\Bpmn\ActivityInterface; ...)